src/App.js

```diff
@@ -3,7 +3,7 @@ import Calculator from './components/calculator/Calculator';
 
 const App = () => (
   <div className="App">
-    <Calculator className="calculator" />
+    <Calculator />
```

Comment:
- [ ] **Optional:** I think the click animation looks good on your numbers and give the feel that a click has been triggered, attempt to add them to the operators and see if you like them.

https://github.com/til2to/math-magicians-setup/blob/adff3da0f2171eb2d52b368e8f9b7486cdac15b7/src/components/calculator/Calculator.js#L20

Reply:
alright